src/common/tests/btree.c

```diff
@@ -1017,7 +1006,16 @@ main(int argc, char **argv)
 	if (rc != 0)
 		return rc;
 
-	optind = 0;
+	if (argc == 1) {
+		print_message("Invalid format.\n");
+		return -1;
+	}
+
+	if(strcmp(argv[1], "--start-test") == 0){
```

Comment:
(style)  space required before the open brace '{'
(style)  space required before the open parenthesis '('